src/main/java/com/xceptance/neodymium/util/Neodymium.java

```diff
@@ -515,4 +533,47 @@ public static String getNeodymiumVersion()
         final String buildVersion = getContext().getClass().getPackage().getImplementationVersion();
         return buildVersion == null ? "?.?.?" : buildVersion;
     }
+
+    /**
+     * Saves the last used locator
+     * 
+     * @param by
+     *            CSS Selector
+     */
+    public static void setLastUsedLocator(By by)
+    {
+        getContext().lastLocator = by;
+        getContext().lastUsedElement = null;
+    }
+
+    /**
+     * Saves the last used locator and the element under which to search for
+     * 
+     * @param element
+     *            Webelement
+     * @param by
+     *            CSS Selector
+     */
+    public static void setLastUsedLocator(WebElement element, By by)
+    {
+        getContext().lastLocator = by;
+        getContext().lastUsedElement = element;
+    }
+
+    /**
+     * Returns the last used Element
+     * 
+     * @return last used Element
+     */
+    public static WebElement getLastUsedElement()
+    {
+        if (getContext().lastUsedElement != null)
+        {
+            return getContext().lastUsedElement.findElement(getContext().lastLocator);
```

Comment:
if the element cannot be found on the page the `NoSuchElementException` is thrown here. Do we want to catch it?